Doc/faq/windows.rst

```diff
@@ -132,10 +132,10 @@ you should make sure that entering the command ::
    c:\Python33\python
 
 starts up the interpreter as above (and don't forget you'll need a ":kbd:`Ctrl-Z`" and
-an ":kbd:`Enter`" to get out of it). Once you have verified the directory, you can
-add it to the system path to make it easier to start Python by just running
-the ``python`` command. This is currently an option in the installer as of
-CPython 3.3.
+an ":kbd:`Enter`" or type ``exit()`` in the interpreter to get out of it).
```

Comment:
I think adding a mention of 'exit()' here just confuses the exposition.  I recommend against this change.